52.md

```diff
@@ -0,0 +1,116 @@
+
+NIP-52
+======
+
+Badges
+------
+
+`draft` `optional` `author:neilck`
+
+*This NIP is currently in draft, and review / comments are sought.* 
+
+This NIP enables an app owner (or community manager) to award of badges to their members. 
+
+## Kind XX: Issue Badge
+
+EVENT sent from client to relay to issue a badge.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the app owner issuing the badge>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": XX,
+  "tags": [
+    ["p", <32-bytes hex pubkey of the member receiving the badge>],
+    ["badge_type", <integer specifying badge type>]
+  ],
+  "content": <stringified json object, matching badge_type schema>,
+  "sig": <64-bytes signature of the sha256 hash of the serialized event data, which is the same as the "id" field>
+}
+```
+
+`p tag` MUST contain exactly one entry, the pubkey of the claim's subject.
+
+`badge_type` identifies badge type. Current accept values are `0 - Achievement Badge` and `1 - Membership Badge`
+
+Like Kind 0, badges provide metadata about a pubkey, but the event is sent (and thus signed) by the app owner, not the subject's pubkey.
+
+Two badge types are defined, with the possibility of adding more if required.
+
+### 0 - Achievement Badge
+Awarded to a user for an achievement. 
+
+Achievement Badge Schema
+```json
+{
+    "issuer_name": <string of app name or community name>,
+    "name": <string of achievement name>,
+    "message": <string of message>,
+    "image": <url of badge image>,
```

Comment:
Hard coded image URLs are fragile. Figure out how to automatically generate based on some data the app owner controls, perhaps stored in Kind: 0 metadata.